rosetta-source/src/main/resources/cdm-sample-files/fpml-5-10/incomplete-products/commodity-derivatives/expectations.json

```diff
@@ -24,7 +24,7 @@
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/commodity-derivatives/com-ex34-gas-put-option-european-floating-strike.xml","excludedPaths":0,"externalPaths":86,"outstandingMappings":30,"validationFailures":9,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/commodity-derivatives/com-ex35-call-option-gas-power-heat-rate-daily.xml","excludedPaths":0,"externalPaths":77,"outstandingMappings":25,"validationFailures":7,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/commodity-derivatives/com-ex36-gas-call-option-european-spread-negative-premium-floating-strike.xml","excludedPaths":0,"externalPaths":79,"outstandingMappings":23,"validationFailures":8,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/commodity-derivatives/com-ex37-gold-forward-offered-rate.xml","excludedPaths":0,"externalPaths":97,"outstandingMappings":1,"validationFailures":0,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_Basis","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/commodity-derivatives/com-ex37-gold-forward-offered-rate.xml","excludedPaths":0,"externalPaths":97,"outstandingMappings":1,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_Basis","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
```

Comment:
Why the extra commodity validation failure?

Reply:
The interest rate spread is in a different `PriceQuantity` to the observable, so the new condition `PriceQuantity.ObservableExists` fails.